server/src/Config.hs

```diff
@@ -111,6 +115,16 @@ configInfo environment = info parser description
     description = fullDesc <>
       header "Icepeak - Fast Json document store with push notification support."
 
+-- * Parsers
+
+storageBackend :: Parser StorageBackend
+storageBackend = fileBackend <|> sqliteBackend
```

Comment:
For backwards compatibility, it may be nice to default to `--file`.

Reply:
Good idea, fixed.

Reply:
I don’t see a change here?

Reply:
I had forgotten to push it (was on a different machine).